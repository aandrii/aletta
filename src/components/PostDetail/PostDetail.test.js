import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import PostDetail from './PostDetail'

test('component', () => {
  const component = shallow(<PostDetail />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
