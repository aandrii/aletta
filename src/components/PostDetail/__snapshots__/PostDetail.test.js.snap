exports[`test component 1`] = `
<div>
  <Post />
</div>
`;
